### ISSUE TITLE
Donate $ to bioconda

### ISSUE THREAD

**Issue by endrebak** [bioconda/bioconda-recipes#2300]
I have more money than time (not much of either though). Would you be able to set up a paypal and use donations for bioconda-related purposes? I can't give much, but I give worthy FOSS projects 5-10 bucks every now and then...

**Comment by tomkinsc** [comment]
That is very kind of you, @endrebak! Paging @johanneskoester and @bgruening. 

Related to this: If bioconda more of a full-time activity for its maintainers, a case could be made for it to operate as an independent non-profit organization (something like a 501(c)(3) in the US). This would make it easier to handle things like donations, and would make it more compelling for companies (like Continuum) to donate resources to the effort (since such donations would be tax deductible). 

Another possible option would be for bioconda to operate officially under the auspices of another non-profit organization (such as one of the many non-profit research institutions associated with our contributors). As an example of a project that does this: PyPy accepts donations via the Software Freedom Conservancy.

**Comment by kyleabeauchamp** [comment]
I think NumFocus is a good model:

http://www.numfocus.org/

**Comment by tomkinsc** [comment]
NumFocus looks great, @kyleabeauchamp, and Continuum is a founding corporate sponsor so bioconda should be right at home there.

**Comment by johanneskoester** [comment]
Indeed, this is a good idea. I will have a look what to do once the build system migration is finally done.

**Comment by bgruening** [comment]
@endrebak This is a great offer thanks a lot! What would be a pretty doable solution in short term is if we could pay travis or any other CI to extend build-time or concurrent jobs.

**Comment by johanneskoester** [comment]
Yes, CI speed is really the area to improve at the moment.
